src/templates/tokens/colortokens.pr

```diff
@@ -1,25 +1,34 @@
 {[ inject "auto_generated_comment" context self /]}
+{[ const requestedGroups = ["Natural", "Primary", "Others", "ADA"] /]}
```

Comment:
We shouldn't have to do this. We should just be able to filter out the Component tokens.

Reply:
Updated! I was having trouble the first time trying to decipher the crazy JSON that was returned from all these `ds` calls. But I've removed the above in favor of a filter function.